### PR TITLE
Convert busy times to use `AppDevHistogram`

### PR DESCRIPTION
**Pull request by kevinchan159**
## Overview

I replaced the old popular busy times histogram with the new `AppDevHistogram` cocoapod. We know also have haptics when the user scrolls through the times! It also seems that something was off with the logic used to display the bars because bars would previously have the same height even if their busyness value was different. This is fixed now though.

## Changes Made

Replaced Popular
Switch from using `Histogram` to new `PopularTimesHistogramView` which displays a `HistogramView`.

## Test Coverage

Ran the app and tested scrolling through the hours!

## Screenshots (delete if not applicable)

Demo video in `#uplift-ios`

### CODE REVIEW

File: Uplift/Views/Histogram.swift [hunk 1]
```diff
@@ -27,7 +27,7 @@ class Histogram: UIView {
     private let highThreshold = 57
     private let mediumThreshold = 25
     private let secondsPerHour: Double = 3600.0
-    private let timeDescriptors = [ClientStrings.Histogram.businessLevel1, ClientStrings.Histogram.businessLevel2, ClientStrings.Histogram.businessLevel3]
+    private let timeDescriptors = [ClientStrings.Histogram.busynessLevel1, ClientStrings.Histogram.busynessLevel2, ClientStrings.Histogram.busynessLevel3]
```
Comment (Omarrasheed) [inline Uplift/Views/Histogram.swift]: high iq

File: Uplift/Views/GymDetail/PopularTimesHistogramView.swift [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        // Get hour of day, i.e. "12p"
+        let hourOfDay = startHourDate.addingTimeInterval(numSecondsPastStartHour)
+            .getStringOfDatetime(format: "ha")
+
+        // Get busyness rating, i.e. "A little busy"
+        let busynessLevelRating = getBusynessLevelDescriptor(for: index)
+
+        let attributedString = NSMutableAttributedString(
+            string: "\(hourOfDay) : \(busynessLevelRating)",
+            attributes: [NSAttributedString.Key.foregroundColor: UIColor.gray04]
```
Comment (whoiswillma) [inline Uplift/Views/GymDetail/PopularTimesHistogramView.swift]: Can use type inference to shorten this to `[.foregroundColor: UIColor.gray04]`
